src/models/document/document-content-tests/dc-shared-models.test.ts

```diff
@@ -264,25 +276,25 @@ Object {
       expect(getSnapshot(targetDocument)).toMatchInlineSnapshot(`
 Object {
   "rowMap": Object {
-    "testid-28": Object {
+    "testid-32": Object {
       "height": undefined,
-      "id": "testid-28",
+      "id": "testid-32",
```

Comment:
FYI, there's an example of a custom jest matcher which handles ids specially in `geometry-content.test.ts` called `toEqualWithUniqueIds` in case you want to consider writing a matcher which is more robust to these kind of id changes at some point.